internal/conf/rate.go

```diff
@@ -0,0 +1,69 @@
+package conf
+
+import (
+	"fmt"
+	"strconv"
+	"strings"
+	"time"
+)
+
+type Rate struct {
+	Events   float64       `json:"events,omitempty"`
+	OverTime time.Duration `json:"over_time,omitempty"`
+}
+
+func (r *Rate) EventsPerSecond() float64 {
+	if int64(r.OverTime) == 0 {
+		return r.Events
+	}
+
+	return r.Events / r.OverTime.Seconds()
+}
+
+// DefaultOverTime sets the OverTime field to overTime if it is 0.
+func (r *Rate) DefaultOverTime(overTime time.Duration) Rate {
```

Comment:
this function doesn't seem to be used anywhere else?

Reply:
Good catch, removed it.